docs/source/buildingWindows.rst

```diff
@@ -0,0 +1,74 @@
+
+Interactively building SimpleITK on Windows
+###########################################
+
+This page describes how to interactively build a SimpleITK Superbuild on
+Windows.  To build SimpleITK we use CMake to generate a Visual Studio project,
+and Visual Studio to compile the C/C++ code.
+
+The Windows version of CMake includes a graphical-user-interface (GUI) version
+of the program, in addition to the standard command-line CMake.  Here is what
+the CMake GUI looks like:
+
+.. image:: ../images/cmake-win-1.png
+  :width: 400
+  :alt: Windows CMake GUI
+
+In the above example, the source directory is 'C:/SimpleITK', and the
+build directory is 'C:/SimpleITK-build'.  It is best to place the source
+and build directories at the root of a volume to minimize path name lengths.
```

Comment:
Possibly explicitly say that windows has a default max path length of 260 characters and ITK has long paths which can cause build issues? https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd